generator/definition.go

```diff
@@ -49,12 +80,19 @@ func (g DefinitionGenerator) CreateDefinition(t interface{}) {
 	}
 }
 
-func (g DefinitionGenerator) createStructDefinitions(_struct reflect.Type) map[string]definition.DefinitionProperties {
+func (g DefinitionGenerator) createStructDefinitions(_struct reflect.Type, data any, prefix string) map[string]definition.DefinitionProperties {
 	properties := make(map[string]definition.DefinitionProperties)
 	for i := 0; i < _struct.NumField(); i++ {
 		field := _struct.Field(i)
 		fieldType := getType(field.Type.Kind().String())
 		fieldJsonTag := getJsonTag(field)
+		exampleTag := getExampleTag(field)
+
+		// check swagno tag for custom type
```

Comment:
we have another json struct tag that we're parsing?

Reply:
There was a pull request:
https://github.com/go-swagno/swagno/pull/18 

What do you think?